tests/cephadm/test_cephadm_ansible_wrapper.py

```diff
@@ -68,6 +69,7 @@ def setup_cluster(ceph_cluster, config):
 
     if not Rpm(installer).query("cephadm-ansible"):
         SetUpSSHKeys.run(installer, nodes)
+        SetupLicence.run(installer)
```

Comment:
This will install for all the ceph version right Vaibhav?

Reply:
@pranavprakash20 yes, it's simple package `ibm-storage-ceph-license` independent of ceph version